tests/pretty_format_java_test.py

```diff
@@ -22,21 +27,45 @@ def undecorate_method():
         yield undecorated
 
 
+@pytest.mark.parametrize(
+    "version",
+    (
+        _get_default_version("google_java_formatter"),
+        # The following explicit versions re neededbecause the format
+        # of the binary URL has changed on release 1.10.0
+        "1.9",
+        "1.10.0",
+    ),
+)
+@pytest.mark.integration
+def test__download_google_java_formatter_jar(ensure_download_possible, version):  # noqa: F811
+    # Test that we can download different version of the Google Java Formatter
+    _download_google_java_formatter_jar(version)
+
+
 @pytest.mark.parametrize(
     ("cli_args", "expected_retval"),
     (
         (["invalid.java"], 1),
         (["pretty-formatted.java"], 0),
         (["not-pretty-formatted.java"], 1),
         (["not-pretty-formatted_fixed.java"], 0),
-        # Test different google-java-formatter versions
-        (["--google-java-formatter-version=1.10.0", "pretty-formatted.java"], 0),
-        (["--google-java-formatter-version=1.9", "pretty-formatted.java"], 0),
     ),
 )
 def test_pretty_format_java(undecorate_method, cli_args, expected_retval):
     assert undecorate_method(cli_args) == expected_retval
 
 
+@pytest.mark.skipif(condition=get_jdk_version() < Version("16"), reason="Skipping test because it requires Java JDK 16+")
+def test_pretty_format_java_up_to_1_9_is_not_allowed_on_jdk_16(undecorate_method):
```

Comment:
```suggestion
def test_pretty_format_java_up_to_1_9_is_not_allowed_on_jdk_16_and_above(undecorate_method):
```

Or change the `condition` to `= Version("16")`

Reply:
The test should be executed only on Java 16+ ... skipif expects condition to be `True` if the test should be skipped and as such not executed